_episodes/05-contribution-governance-communication.md

```diff
@@ -1,52 +1,380 @@
+> - [Py4J - A Bridge between Python and Java](https://www.py4j.org/contributing.html)
+> - [JuliaLang](https://github.com/JuliaLang/julia/blob/master/CONTRIBUTING.md)
+> - [dplyr](https://github.com/tidyverse/dplyr/blob/master/.github/CONTRIBUTING.md)
+> - [octobox](https://github.com/octobox/octobox#contribute)
+> - [scipy](https://github.com/scipy/scipy/blob/master/CONTRIBUTING.rst)
+> - [plotly](https://github.com/ropensci/plotly/blob/master/CONTRIBUTING.md)
+> - [scrubr](https://github.com/ropensci/scrubr/blob/master/CONTRIBUTING.md)
+> - [rOpenSci](https://github.com/ropensci/dev_guide/blob/master/maintenance_contributing.Rmd)
+> - Ubuntu Contributing Guide: [short version](https://discourse.ubuntu.com/t/contribute/26) and [longer version](https://wiki.ubuntu.com/ContributeToUbuntu).
+> - Short example [sf](https://github.com/r-spatial/sf#contributing)
+> - [knitr](https://github.com/yihui/knitr/blob/master/.github/CONTRIBUTING.md)
+> - [rmarkdown](https://github.com/rstudio/rmarkdown/blob/master/CONTRIBUTING.md)
 {: .challenge}
 
+### Support your contributors!
 
-## Communication
+As your project grows and more people start to contribute, in addition to the
+minimal CONTRIBUTING.md file you might need to provide support to your
+contributors in order to encourage them to make future contributions.
+For example, it's important to identify what motivates developers to contribute,
+what resources they need to work on your project (e.g.codes, data, metadata),
+what are the right skills to work on a certain task, and if they lack any skill,
+they have resources to help them develop those skills.
+
+### Reward your contributors!
+
+As an author, you should acknowledge all your contributors and reward their
+contributions for example, by mentioning them in your contributors list,
+by offering them opportunities for training, teaching or mentoring,
+or list them as co-authors if the contribution is substantial. This can be explained
+in a simple paragraph in the CONTRIBUTING.md file or extensive in an separate document
+depending on the size of your project and team.
+
+> ## Discussion: Supporting your contributors
+> #### Time: 5 minutes
+> Think about the different resources that your user might need, in
+> order to make a contribution. Also think at what stages these
+> resources might be needed.
+> Discuss in groups and list the resources.
+>
+> > ## Solution
+> >
+> > Here is a list of recommendations to provide support to your
+> > contributors, but remember this is not an extensive list.
+> > - Installation instructions
+> > - Style guidelines
+> > - FAQ’s
+> > - Code review process
+> > - Resources to develop skills required to contribute to the project
+> > - Community calls / meetings
+> > - Archive
+> {: .solution}
+{: .discussion}
+
+### Promote your work
+
+For the developers to make relevant contributions, it is also worth thinking
```

Comment:
This is a very long and hard to understand sentence. Can you rewrite and simplify it?